### ISSUE TITLE
Global step time is fluctuating randomly between steps.

### ISSUE THREAD

**Issue by kerolos** [tensorflow/models#3000]
# Describe the problem:

I used this model "ssd_mobilenet_v1_coco_11_06_2017" to be able to classify and localize my own "3 objects". My problem is that the global step takes sometimes between 0.5 sec/step to 2 sec /step and sometimes takes 400sec/step to 1200 sec/ step. the computer finished only 700 steps within 20 Hours, which is really weird.
Also, something I realized, 97% of the Ram is occupying when Recording summary step and the following global step occurred.

# System information:

OS: Ubuntu desktop 14.04. 
Ram: 16 GB.
Nvidia: GTX 1080  TI 11 GB ram.
Tensorflow: v1.4.0-rc1-11-g130a514.
batch-size: 10 .
CUDA/cuDNN version: Cuda 8.0 and cudnn 6 .

# Output of training processes:

INFO:tensorflow:Recording summary at step 664.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 665.
INFO:tensorflow:global step 665: loss = 1.4688 (646.632 sec/step)
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 666: loss = 1.7181 (195.437 sec/step)
INFO:tensorflow:global step 667: loss = 2.0240 (2.869 sec/step)
INFO:tensorflow:global step 668: loss = 2.0866 (14.212 sec/step)
INFO:tensorflow:global step 669: loss = 2.1031 (2.734 sec/step)
INFO:tensorflow:global step 670: loss = 1.5433 (2.310 sec/step)
INFO:tensorflow:global step 671: loss = 2.0120 (2.487 sec/step)
INFO:tensorflow:global step 672: loss = 2.0841 (1.780 sec/step)
INFO:tensorflow:Recording summary at step 672.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 673: loss = 1.3150 (248.055 sec/step)
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 674.
INFO:tensorflow:global step 674: loss = 1.4747 (245.788 sec/step)
INFO:tensorflow:Recording summary at step 674.
INFO:tensorflow:global step 675: loss = 2.3231 (244.705 sec/step)
INFO:tensorflow:global step 676: loss = 2.8910 (10.194 sec/step)
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 677: loss = 1.3781 (55.264 sec/step)
INFO:tensorflow:global step 678: loss = 1.7793 (34.336 sec/step)
INFO:tensorflow:global step 679: loss = 1.4643 (18.670 sec/step)
INFO:tensorflow:global step 680: loss = 1.7935 (1.683 sec/step)
INFO:tensorflow:global step 681: loss = 2.5205 (1.933 sec/step)
INFO:tensorflow:global step 682: loss = 1.9633 (2.103 sec/step)
INFO:tensorflow:global step 683: loss = 1.4078 (0.795 sec/step)
INFO:tensorflow:global step 684: loss = 1.8395 (1.746 sec/step)
INFO:tensorflow:global step 685: loss = 1.5991 (1.230 sec/step)
INFO:tensorflow:Recording summary at step 685.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 686.
INFO:tensorflow:global step 686: loss = 1.6328 (1321.527 sec/step)
INFO:tensorflow:Recording summary at step 686.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 686.
INFO:tensorflow:Recording summary at step 687.
INFO:tensorflow:global step 687: loss = 1.8285 (478.037 sec/step)
INFO:tensorflow:Recording summary at step 687.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 688: loss = 2.2072 (46.042 sec/step)
INFO:tensorflow:global step 689: loss = 1.7111 (17.323 sec/step)
INFO:tensorflow:global step 690: loss = 1.7003 (1.249 sec/step)
INFO:tensorflow:global step 691: loss = 1.7168 (2.057 sec/step)
INFO:tensorflow:global step 692: loss = 2.2428 (1.512 sec/step)
INFO:tensorflow:global step 693: loss = 2.1942 (1.331 sec/step)
INFO:tensorflow:global step 694: loss = 1.6748 (0.981 sec/step)
INFO:tensorflow:global step 695: loss = 1.8890 (1.642 sec/step)
INFO:tensorflow:global step 696: loss = 2.7845 (1.410 sec/step)
INFO:tensorflow:global step 697: loss = 1.9228 (1.421 sec/step)
INFO:tensorflow:Recording summary at step 697.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 698.
INFO:tensorflow:global step 698: loss = 1.5280 (1297.641 sec/step)
INFO:tensorflowINFO:tensorflow:Recording summary at step 664.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 665.
INFO:tensorflow:global step 665: loss = 1.4688 (646.632 sec/step)
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 666: loss = 1.7181 (195.437 sec/step)
INFO:tensorflow:global step 667: loss = 2.0240 (2.869 sec/step)
INFO:tensorflow:global step 668: loss = 2.0866 (14.212 sec/step)
INFO:tensorflow:global step 669: loss = 2.1031 (2.734 sec/step)
INFO:tensorflow:global step 670: loss = 1.5433 (2.310 sec/step)
INFO:tensorflow:global step 671: loss = 2.0120 (2.487 sec/step)
INFO:tensorflow:global step 672: loss = 2.0841 (1.780 sec/step)
INFO:tensorflow:Recording summary at step 672.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 673: loss = 1.3150 (248.055 sec/step)
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 674.
INFO:tensorflow:global step 674: loss = 1.4747 (245.788 sec/step)
INFO:tensorflow:Recording summary at step 674.
INFO:tensorflow:global step 675: loss = 2.3231 (244.705 sec/step)
INFO:tensorflow:global step 676: loss = 2.8910 (10.194 sec/step)
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 677: loss = 1.3781 (55.264 sec/step)
INFO:tensorflow:global step 678: loss = 1.7793 (34.336 sec/step)
INFO:tensorflow:global step 679: loss = 1.4643 (18.670 sec/step)
INFO:tensorflow:global step 680: loss = 1.7935 (1.683 sec/step)
INFO:tensorflow:global step 681: loss = 2.5205 (1.933 sec/step)
INFO:tensorflow:global step 682: loss = 1.9633 (2.103 sec/step)
INFO:tensorflow:global step 683: loss = 1.4078 (0.795 sec/step)
INFO:tensorflow:global step 684: loss = 1.8395 (1.746 sec/step)
INFO:tensorflow:global step 685: loss = 1.5991 (1.230 sec/step)
INFO:tensorflow:Recording summary at step 685.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 686.
INFO:tensorflow:global step 686: loss = 1.6328 (1321.527 sec/step)
INFO:tensorflow:Recording summary at step 686.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 686.
INFO:tensorflow:Recording summary at step 687.
INFO:tensorflow:global step 687: loss = 1.8285 (478.037 sec/step)
INFO:tensorflow:Recording summary at step 687.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 688: loss = 2.2072 (46.042 sec/step)
INFO:tensorflow:global step 689: loss = 1.7111 (17.323 sec/step)
INFO:tensorflow:global step 690: loss = 1.7003 (1.249 sec/step)
INFO:tensorflow:global step 691: loss = 1.7168 (2.057 sec/step)
INFO:tensorflow:global step 692: loss = 2.2428 (1.512 sec/step)
INFO:tensorflow:global step 693: loss = 2.1942 (1.331 sec/step)
INFO:tensorflow:global step 694: loss = 1.6748 (0.981 sec/step)
INFO:tensorflow:global step 695: loss = 1.8890 (1.642 sec/step)
INFO:tensorflow:global step 696: loss = 2.7845 (1.410 sec/step)
INFO:tensorflow:global step 697: loss = 1.9228 (1.421 sec/step)
INFO:tensorflow:Recording summary at step 697.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:Recording summary at step 698.
INFO:tensorflow:global step 698: loss = 1.5280 (1297.641 sec/step)
INFO:tensorflow:Recording summary at step 698.
INFO:tensorflow:global step 699: loss = 1.4803 (51.523 sec/step)
INFO:tensorflow:Recording summary at step 699.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 700: loss = 1.9215 (416.182 sec/step)
INFO:tensorflow:Recording summary at step 700.

:Recording summary at step 698.
INFO:tensorflow:global step 699: loss = 1.4803 (51.523 sec/step)
INFO:tensorflow:Recording summary at step 699.
INFO:tensorflow:Saving checkpoint to path training/model.ckpt
INFO:tensorflow:global step 700: loss = 1.9215 (416.182 sec/step)
INFO:tensorflow:Recording summary at step 700.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
